src/components/listings/MapLocationListing.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import {
+  ImageBackground,
+  View,
+  Text,
+  Pressable,
+  GestureResponderEvent,
+} from 'react-native'
+import { MapLocationListingItem } from '@/types/location'
+
+interface MapLocationListingProps {
+  locations: MapLocationListingItem[]
+  onPress: (event: GestureResponderEvent) => void
+}
+
+function MapLocationListing(props: MapLocationListingProps) {
+  const { locations, onPress } = props
+
+  return (
+    <View className="flex flex-col items-center gap-2">
+      {locations.map((location) => (
+        <Pressable
+          key={location.name}
+          onPress={onPress}
```

Comment:
then we can use `() => onPress(location)`?

Reply:
👍